ActindoWorkFlow/Operation/01_ManageWorkflows.md

```diff
@@ -0,0 +1,423 @@
+[!!Workflows](ActindoWorkFlow)
+
+# Manage the workflows
+
+The *Workflows* module allows the customers to define their own business processes, such as orders, shipments, returns, etc.
+The workflows can be created individually in the workflow editor, they can be edited subsequently and by publishing and unpublishing a workflow, the usage can be controlled. Further, obsolete workflows can be deleted.
```

Comment:
```suggestion
The workflows can be created individually in the workflow editor, they can be edited subsequently and by publishing and unpublishing a workflow, the usage can be controlled. Further, obsolete workflows can be deleted.
```

Stimmt der letzte Satz? Ich habe gedacht in dieser Version kann man noch nichts löschen.